src/css/imports/mixins.less

```diff
@@ -396,20 +400,6 @@
                     }
                 }
             }
-
-            // When we're in an error or buffering state...
-            &.jw-error,
-            &.jw-state-error,
-            &.jw-state-buffering {
-                // Show the normal state for the icons so we do not imply its clickable
-                .jw-display-icon-container {
-                    background-color: @display-bkgd-color;
```

Comment:
@jw-kaurand this was making the bg color of the buffering icon gray in skins because it's more specific than the skin color rules. When we don't want something to happen, don't override, omit.